posts/2016-12-15-serverless-v1.4.0.md

```diff
@@ -0,0 +1,146 @@
+---
+title: C# support, CLI based plugin discovery and management, Alexa event support, service wide metrics in Serverless Framework v1.4
+description: C# support, CLI based plugin discovery and management, Alexa event support, service wide metrics in Serverless Framework v1.4
+date: 2016-12-15
+layout: Post
+---
+
+We're happ to release v1.4.0 of the Serverless Framework today!
+
+This release is packed with a lot of great, new feature. Let's take a look at the highlights!
+
+## Highlights of 1.4.0
+
+Here are the highlights of this release. You can find a list with all the updates in the [changelog](https://github.com/serverless/serverless/blob/master/CHANGELOG.md).
+
+### CLI based plugin discovery and management
+
+Plugins are a nice way to extend Serverless with 3rd party functionality.
+
+Do you need `webpack` support? There's [a plugin](https://github.com/elastic-coders/serverless-webpack) for that!
+
+Do you want offline support? There's [a plugin](https://github.com/dherault/serverless-offline) for that!
+
+Recently we've introduced the [Serverless plugin registry](https://github.com/serverless/plugins) which is a GitHub repository containg a list of all the available Serverless plugins.
+
+Serverless now integrates with this repository so you can discover, install and uninstall plugins in your service!
+
+Here's what it looks like:
+
+#### List all available plugins:
+
+```bash
+serverless plugin list
+```
+
+#### Search for a plugin
+
+```
+serverless plugin search --query webpack
+```
+
+#### Install a plugin
+
+```bash
+serverless plugin install --name serverless-webpack
+```
+
+#### Uninstall a plugin
+
+```bash
+serverless plugin uninstall --name serverless-webpack
+```
+
+This makes it a lot easier to find and manage Serverless plugins for your service
+
+**Note:** You're a plugin author or found a Serverless plugin which is misisng? Feel free to submit a PR to the [Serverless plugin registry](https://github.com/serverless/plugins/edit/master/plugins.json)!
+
+### Alexa event support
+
+v1.4 also introduces a new event `alexaSkill` event source.
+
+You can now hook up your Lambdas with `alexaSkill` events like this:
+
+```yml
+functions:
+  hello:
+    handler: handler.hello
+    events:
+      - alexaSkill: true
+```
+
+You can read more in [the docs](https://serverless.com/framework/docs/providers/aws/events/alexa-skill) about this new event source.
+
+### C# service template
+
+AWS introduced Dotnet core support for Lambdas at re:Invent.
+
+Serverless has you covered and ships with the new `aws-csharp` template you can use to create C# services on AWS Lambda.
```

Comment:
we should add link to https://github.com/serverless/serverless/tree/master/docs/providers/aws/examples/hello-world/csharp somwhere here. C# requires some dotnet tools to run before `sls deploy`